src/backend/marsha/core/migrations/0027_device.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.9 on 2021-11-23 14:11
```

Comment:
As you want, but the idea here was to separate commits nicely.
But I can group them in the first one, just let me know.

Reply:
Looking at the 2 commits I think there should be only one commit since it is the same feature. One e 
Ndpoint does not make sense without the other one.